### ISSUE TITLE
Test FreePI / JSONPlaceholder Package by developing an application using it

### ISSUE THREAD

**Issue by AlexGaiser** [AlexGaiser/freepi#38]

**Comment by AlexGaiser** [comment]
We should make sure we create sample applications that include a typescript react application, JS react application, Node, TypeScript Node, vanilla JS, etc. 

Current sample applications are housed at the following repository: 
https://github.com/AlexGaiser/freepi-sample-apps

**Comment by AlexGaiser** [comment]
We may consider tagging this and asking for contributions using the Package

**Comment by AlexGaiser** [comment]
We should make sure we create sample applications that include a typescript react application, JS react application, Node, TypeScript Node, vanilla JS, etc. 

Current sample applications are housed at the following repository:

**Comment by AlexGaiser** [comment]
I am working on developing a basic typescript react application. 
I found that parameters for the query on the "find" method are not being included. I have made changes to the Type of the "params" parameter which should account for this. This begs the question: should we limit the 'params' object in the find method to only being an object of the shape it is expected to return? IE Todo has params object of shape "Todo"? I cannot think of a circumstance where you would search JSONPlaceholder using param properties that are not on the object you are searching for
